R/util.r

```diff
@@ -7,7 +7,7 @@
 #' @keywords internal
 # @param short  Whether to use unqualified host name (before first dot)
 host = function(node=getOption("clustermq.host", Sys.info()["nodename"]),
-                ports=6000:9999, n=100) {
+                ports=getOption("clustermq.port", 6000:9999), n=100) {
     utils::head(sample(sprintf("tcp://%s:%i", node, ports)), n)
```

Comment:
If less than 100 ports are specified now, the `sample` on the next line with fail with:

```r
Error in sample.int(length(x), size, replace, prob) :
  cannot take a sample larger than the population when 'replace = FALSE'
```

So on L11, instead of `n` it should sample `min(n, length(ports))`

Reply:
Ah please ignore, I understood my own code wrong. It's fine like it is!